src/bridge/cockpitdbusjson.c

```diff
@@ -122,6 +122,30 @@ check_type (JsonNode *node,
   return TRUE;
 }
 
+static gboolean
+check_int_type (JsonNode *node,
+                const char *type_name,
```

Comment:
As this is only ever being used for checking `GVariant` ranges, how about passing the `type` field directly and using `g_variant_get_type_string()` on it, instead of manually creating the type strings?